### PR TITLE
[MediaController] Add the result code for Response and playback state enum

### PR DESCRIPTION
**Pull request by hsgwon**
### Description of Change ###

Add the result code for Response and playback state enum

### API Changes ###

 - TCSACR-340

### CODE REVIEW

File: src/Tizen.Multimedia.Remoting/MediaController/MediaControlEnums.cs [hunk 1]
```diff
@@ -276,6 +294,88 @@ public enum MediaControlDisplayMode
...
+        /// </summary>
+        NoPeer,
+
+        /// <summary>
+        /// The network has failed.
+        /// </summary>
+        NetworkFailed = 500,
+
+        /// <summary>
+        /// Cropped full screen
```
Comment (aferin) [inline src/Tizen.Multimedia.Remoting/MediaController/MediaControlEnums.cs:364]: I think the summary should be fixed.

File: src/Tizen.Multimedia.Remoting/MediaController/MediaControlEnums.cs [hunk 2]
```diff
@@ -276,6 +294,88 @@ public enum MediaControlDisplayMode
...
+        /// </summary>
+        NoAccount = 600,
+
+        /// <summary>
+        /// Cropped full screen
```
Comment (aferin) [inline src/Tizen.Multimedia.Remoting/MediaController/MediaControlEnums.cs:369]: Same as above

File: src/Tizen.Multimedia.Remoting/MediaController/MediaControlEnums.cs [hunk 3]
```diff
@@ -276,6 +294,88 @@ public enum MediaControlDisplayMode
...
+        /// </summary>
+        LoginFailed,
+
+        /// <summary>
+        /// Cropped full screen
```
Comment (aferin) [inline src/Tizen.Multimedia.Remoting/MediaController/MediaControlEnums.cs]: Same as above

File: src/Tizen.Multimedia.Remoting/MediaController/MediaControlEnums.cs (hunk 1 above)
Comment (hsgwon) [reply src/Tizen.Multimedia.Remoting/MediaController/MediaControlEnums.cs:364]: I'll fix it.

File: src/Tizen.Multimedia.Remoting/MediaController/MediaControlEnums.cs (hunk 2 above)
Comment (hsgwon) [reply src/Tizen.Multimedia.Remoting/MediaController/MediaControlEnums.cs:369]: I'll fix it.

File: src/Tizen.Multimedia.Remoting/MediaController/MediaControlEnums.cs [hunk 4]
```diff
@@ -276,6 +294,88 @@ public enum MediaControlDisplayMode
...
+
+        /// <summary>
+        /// The command or the event had already been completed.
+        /// </summary>
+        AlreadyDone = 200,
+
+        /// <summary>
+        /// The command or the event is aborted by some external event (e.g. aborted play command by incoming call).
+        /// </summary>
+        Aboarted = 300,
```
Comment (aferin) [inline src/Tizen.Multimedia.Remoting/MediaController/MediaControlEnums.cs:316]: Could you check if spelling is right?
Didn't you mean "Aborted"?

Comment (hsgwon) [reply src/Tizen.Multimedia.Remoting/MediaController/MediaControlEnums.cs:316]: I'll fix it.

**Pull request merged**